xlsx/src/import/worksheets/worksheet_streaming.rs

```diff
@@ -0,0 +1,1049 @@
+use crate::error::XlsxError;
+use crate::import::colors::{self, get_indexed_color};
+use quick_xml::events::{BytesStart, BytesText, Event};
+use std::borrow::Cow;
+use std::{
+    collections::HashMap,
+    io::{BufReader, Read},
+};
+
+use ironcalc_base::{
+    expressions::utils::column_to_number,
+    types::{Cell, Col, Row, SheetData, Table, Worksheet, WorksheetView},
+};
+
+use super::{
+    from_a1_to_rc, get_cell_from_excel, get_column_from_ref, get_formula_index,
+    parse_cell_reference, parse_range, SheetSettings, SheetView,
+};
+
+pub(super) fn load_sheet<R: Read + std::io::Seek>(
+    archive: &mut zip::read::ZipArchive<R>,
+    path: &str,
+    settings: SheetSettings,
+    worksheets: &[String],
+    tables: &HashMap<String, Table>,
+    shared_strings: &mut Vec<String>,
+) -> Result<(Worksheet, bool), XlsxError> {
+    let mut sheet_parser = SheetParser::new(settings, shared_strings, worksheets, tables);
+
+    let zipfile = archive.by_name(path)?;
+    let xmlfile = BufReader::new(zipfile);
+    let mut xmlfile = quick_xml::Reader::from_reader(xmlfile);
+    xmlfile.config_mut().expand_empty_elements = true;
+
+    const BUF_SIZE: usize = 700;
```

Comment:
Empirically determined. This was the max element size in the xlsx I was testing. If it's too small, not a big deal as quick_xml will enlarge it if needed.